awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -8,6 +8,7 @@ import {
   Form,
   FormGroup,
   Card,
+  CardBody,
```

Comment:
We can remove Card and CardBody imports now that we're not using them.

Reply:
👍🏻